client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -93,10 +102,24 @@ define([
                 mainItem: post.groups[1].refs[0],
                 items: post.groups[1].refs
             });
-            // complete Post With User Information
-            userList.getUser(post.original_creator).then(function(user) {
-                post.user = user;
+            post.comments = _.reduce(post.groups[1].refs, function(is, val) {
+                return is || _.isUndefined(val.item.name);
+            }, false);
+            // check if `fullDetails` flag is needed
+            // comments items set falg to true.
+            post.fullDetails = post.comments;
```

Comment:
`fullDetails` it is used to check if the post needs information on item.
`comments` it is used to see if the post has comments on it.

`comments` flag it is also used to compile if it is needed the *update time*